src/release/breaking-changes/dispose.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Added missed `dispose()` for some disposable objects in Flutter
+description: >
+  'dispose()' may fail because of double disposal.
+---
+
+## Summary
+
+Missed 'dispose()' are added for some disposable objects. If some other code
+also invokes 'dispose()' for the objects, and the object is protected from double
+disposal, the second 'dispose()' will fail with error message:
```

Comment:
**chore (if-minor):** Use [semantic breaks](https://sembr.org/).
```suggestion
Missed 'dispose()' are added for some disposable objects.
If some other code also invokes 'dispose()' for the objects,
and the object is protected from double disposal,
the second 'dispose()' fails with the following error message:
```

Reply:
Please fix the grammar:
Missed 'dispose()' are added for some disposable objects.
If some other code also invokes 'dispose()' for the objects,

=>
Missing calls to `dispose()` were added for some disposable objects.
If `dispose()` is called elsewhere on the same object